src/twisted/internet/test/reactormixins.py

```diff
@@ -27,14 +30,14 @@
 
 from twisted.python import log
 from twisted.python.failure import Failure
-from twisted.python.compat import _PY3
 
 
 # Access private APIs.
-if platform.isWindows():
-    process = None
-else:
+try:
     from twisted.internet import process
+except ImportError:
+    process = None  # type: ignore
```

Comment:
Could you file a ticket for a more mypy-friendly way of doing this, and link it here before landing?  I am OK with adding `type: ignore` for an import hack like this (particularly in a test helper) but before the pattern replicates ad infinitum I would like to record the fact that we should clean it up somehow later.

Reply:
There already is an open issue https://github.com/python/mypy/issues/1297#issuecomment-568663999 where this use-case was mentioned.  I'll follow up there, and see what I can find out.

Reply:
So based on this: https://github.com/python/mypy/issues/1297#issuecomment-508593494
I think we can remove the `#type: ignore`, and do this:

```
try
    from twisted.internet import process as _process
except ImportError:
    process = None  # type: ignore
else:
    process = _process
```